src/agentic/actor_agents.py

```diff
@@ -747,7 +801,50 @@ def render_events():
             for event in self.next_turn(prompt.prompt):
                 yield (str(event))
         return EventSourceResponse(render_events())
-
+    
+    @app.post("/webhook/{agent_name}/{run_id}/{callback_name}")
```

Comment:
If you look here: https://github.com/supercog-ai/agentic/blob/main/src/agentic/actor_agents.py#L900

the agent name is already the base part of the FastAPI endpoint. So we don't need the agent_name here.